cfme/tests/ssui/test_ssui_service_dialog.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+from cfme.services.service_catalogs import ServiceCatalogs
+from cfme.utils.appliance import ViaSSUI
+from cfme.utils.appliance.implementations.ssui import navigate_to
+
+
+@pytest.fixture(scope="function")
+def tagcontrol_dialog(appliance):
+    service_dialog = appliance.collections.service_dialogs
+    dialog = "dialog_{}".format(fauxfactory.gen_alphanumeric())
+    element_data = {
+        'element_information': {
+            'ele_label': "Service Level",
+            'ele_name': "service_level",
+            'ele_desc': "service_level_desc",
+            'choose_type': "Tag Control",
+        },
+        'options': {
+            'field_category': "Service Level",
+            'field_required': True
+        }
+    }
+    sd = service_dialog.create(label=dialog, description="my dialog")
+    tab = sd.tabs.create(tab_label='tab_{}'.format(fauxfactory.gen_alphanumeric()),
+        tab_desc="my tab desc")
+    box = tab.boxes.create(box_label='box_{}'.format(fauxfactory.gen_alphanumeric()),
+        box_desc="my box desc")
+    box.elements.create(element_data=[element_data])
+    yield sd
+
+
+@pytest.fixture(scope="function")
+def catalog(appliance):
+    catalog = "cat_" + fauxfactory.gen_alphanumeric()
+    cat = appliance.collections.catalogs.create(name=catalog, description="my catalog")
+    yield cat
+
+
+@pytest.fixture(scope="function")
+def catalog_item(appliance, tagcontrol_dialog, catalog):
+    item_name = fauxfactory.gen_alphanumeric()
```

Comment:
Optional:
If you are not using `name` multiple time then better to use directly rather than variable declaration.